Realm/Realm/Linq/TypeSystem.cs

```diff
@@ -63,7 +63,7 @@ private static Type FindIEnumerable(Type seqType)
             var ifaces = seqType.GetInterfaces().ToArray();
             if (ifaces != null && ifaces.Length > 0)
             {
-                foreach (Type iface in ifaces)
+                foreach (var iface in ifaces)
```

Comment:
Unrelated, but I think our unwritten convention would be to use names `@interface` and `interfaces`?